ros2_package_template/include/ros2_package_template/ros2_package_template_node.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef ROS2_PACKAGE_TEMPLATE_H
+#define ROS2_PACKAGE_TEMPLATE_H
```

Comment:
If we follow established practices, this should be named similar to the file name. Also, ROS2 developers are using a trailing underscore in their packages.

In addition, we should discuss if we maybe want to go for `#pragma once` include guards instead of `#ifndef`. We did that switch for Rock in the past.

```suggestion
#ifndef ROS2_PACKAGE_TEMPLATE_NODE_HPP_
#define ROS2_PACKAGE_TEMPLATE_NODE_HPP_
```